documentation/src/main/asciidoc/topics/ref_indexing_configuration.adoc

```diff
@@ -0,0 +1,84 @@
+[id='indexing_configuration-{context}']
+= Index Configuration
+{brandname} configuration controls how indexes are stored and constructed.
+
+== Index Storage
+
+You can configure how {brandname} stores indexes:
+
+* On the host file system, which is the default and persists indexes between restarts.
+* In JVM heap memory, which means that indexes do not survive restarts. +
+You should store indexes in JVM heap memory only for small datasets.
+
+.File system
+[source,xml,options="nowrap",subs=attributes+]
+----
+<distributed-cache name="my-cache">
+   <indexing storage="filesystem" path="${java.io.tmpdir}/baseDir">
+     ...
+   </indexing>
+</distributed-cache>
+----
+
+.JVM heap memory
+[source,xml,options="nowrap",subs=attributes+]
+----
+<distributed-cache name="my-cache">
+  <indexing storage="local-heap">
+   ...
+</distributed-cache>
+----
+
+== Index Reader
+
+The index reader is an internal process that gets updates to the index as it changes over time.
```

Comment:
```suggestion
The index reader is an internal component that provides access to the indexes to perform queries. As the index content changes, {brandname} needs to refresh the reader so that search results are up to date.
```